### DISCUSSION TITLE
Huge difference in assembly stats between metaSPAdes and metaviralSPAdes when assembling gut viromes

### DISCUSSION THREAD

**Discussion by ilnamkang** [ablab/spades#1100] [Q&A] [answered]
Hi,

I performed assembly of my gut viromes uisng both metaSPAdes and metaviralSPAdes.

It seems that the two pipelines give very different results.

Below is the summary stats of the contigs.fasta files I got.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ikang/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ikang/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

Sample1 | No. of contigs | Total length of contigs | mininum length | average length | maximum length
-- | -- | -- | -- | -- | --
metaSPAdes | 90,009 | 61,672,558 | 128 | 685.2 | 307,715
metaviralSPAdes | 98 | 3,119,519 | 2,146 | 31,831.80 | 181,094

</body>

</html>

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ikang/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ikang/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

Sample2 | No. of contigs | Total length of contigs | mininum length | average length | maximum length
-- | -- | -- | -- | -- | --
metaSPAdes | 96,334 | 72,584,866 | 128 | 753.5 | 168,431
metaviralSPAdes | 100 | 3,042,634 | 1,225 | 30,426.30 | 103,608

</body>

</html>

There are huge differences, except for the maximum length of contigs.

Could there be huge differences like these between metaSPAdes and metaviralSPAdes?

My virome sequences were obtained from fecal samples through procedures including viral particle enrichment.

Taxonomic profiling using Phanta (https://www.biorxiv.org/content/10.1101/2022.08.05.502982v1, https://github.com/bhattlab/phanta) showed that ~90% of reads are from viruses.

Thanks.

**Comment by asl** [comment]
Tagging @Dmitry-Antipov

**Comment by Dmitry-Antipov** [comment, marked as answer]
Hi,
metaviralSPAdes was designed to extract viral part of regular metagenomes, so for viral enriched datasets I'd suggest to use metaspades.
Difference between metaviral and meta is expected, since metaviral does not output anything but some paths, suspcicious to be complete viruses (using coverage and graph topology), and metaviral outputs all the assembled sequence, including also fragmented viral genomes, bacterial chromosomes and whatever you sequenced.

**Reply by ilnamkang** [reply]
Thank you for your explanation.
